libstorage/ZdbStorage.cpp

```diff
@@ -108,15 +107,31 @@ void ZdbStorage::SetSqlAccess(SQLBasicAccess::Ptr _sqlBasicAcc)
 
 size_t ZdbStorage::commit(h256 _hash, int64_t _num, const std::vector<TableData::Ptr>& _datas)
 {
-    int32_t rowCount = m_sqlBasicAcc->Commit(_hash, (int32_t)_num, _datas);
-    if (rowCount < 0)
+    volatile int32_t rowCount = 0;
```

Comment:
why volatile

Reply:
https://stackoverflow.com/questions/2024933/warning-might-be-clobbered-on-c-object-with-setjmp